doc/examples/puppeteer/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "axe-puppeteer",
```

Comment:
Should we add a node version here?

Reply:
No, I don't think so. It's specified in the README which is what people are going to see on GitHub.